### DISCUSSION TITLE
Beispiel für Karte mit Marker Darstellung

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/datalinq-community#2] [Q&A]
Liebes Forum,
ich möchte eine Karte (Basemap) mit Markern der Seen Vorarlbergs darstellen. Dazu würde ich ein Beispiel für die Anwendung von @DLH.BeginGeoElementFor mit @DLH.Map benötigen (meine Versuche haben bislang nicht das gewünschte Ergebnis gezeigt).
Als Ergebnis der DataLinq Query erhalte ich eine Liste der Seen und die dazugehörigen Koordinaten (geometry=simple).
Besten Dank!

**Comment by thomaskofler** [comment]
Hallo Jürgen,

also hier wäre meine Lösung zum Einbinden der Karte:

Query:

SELECT
Attribut1
,Attribut2
,Attribut3
,Attribut4
SHAPE.STAsText() as _location,    -- für das Einbinden eines Punkt-Datensatzes
-- SHAPE.STConvexHull().STCentroid().STAsText() as _location --> für das Einbinden einer Linie oder Fläche
SHAPE.STSrid as _location_srid

FROM datenbank.tabelle

**Reply by thomaskofler** [reply]
Und dann kommt die View:

<html>
    <head>
        @{ 
        var mapServices = "orthocacheaktuell";              @*CMS-Url von deiner Hintergrundkarte*@
        var mapExtent = "sbg_m31";          @*CMS-URL Kartenausschnitt *@
        }
    </head>

    <body>
        <table>
            <tr>
                <th>Spalte1</th>
                <th>Spalte2</th>
                <th>Spalte3</th>
                <th>Spalte4</th>
            </tr>
        
            @foreach(var record in Model.Records){
                @DLH.BeginGeoElementFor(record, tag: "tr" 
                                                ,htmlAttributes: new {@class="info" } 
                                                ,popup: "<b>Gemeindegebiet: </b>" + @record["Gemeinde"] + "<b><br />GEMNR: </b>" +                    @record["GEMNR"]
                                                )
                    <td>@record["GEMNR"]</td> 
                    <td>@record["Gemeinde"]</td> 
                    <td>@record["ORT"]</td>
                    <td>@record["Bezirk"]</td>
            @DLH.EndGeoElement(tag: "tr")
        }
        </table>
        
        <br />

            <div>
                @DLH.Map(mapServices, mapExtent, presentations:"",
                htmlAttributes: new { style="width:75%;height:750px", @class="new_class" } )
            </div>
</body>
</html>

**Reply by thomaskofler** [reply]
das hat es jetzt leider etwas falsch dargestellt:
<head>
@{ var mapServices = "orthocacheaktuell"; @*CMS-Url von deiner Hintergrundkarte*@ var mapExtent = "sbg_m31"; @*CMS-URL Kartenausschnitt *@ } 
</head>

**Reply by faulgurke** [reply]
Habe da viel zu kompliziert gedacht. Schönes, einfaches Beispiel.
Vielen Dank für die Hilfe!

**Reply by eni-mira** [reply]
Hello, super! Ich habe dieses Beispiel jetzt auch unter examples/views/table_with_map.cshtml ergänzt!
Im examples Folder könnt ihr zukünftig gerne auch selber Beispiele hinzufügen. :)
LG Mira
